Scalar.FunctionalTests/Tests/EnlistmentPerFixture/CommitGraphStepTests.cs

```diff
@@ -58,10 +62,23 @@ public void CleansUpOphanedLockFiles()
 
             this.fileSystem.CreateEmptyFile(graphLockPath);
 
-            this.Enlistment.RunVerb("commit-graph");
+            this.RunCommitGraphTask();
 
-            this.fileSystem.FileExists(graphLockPath).ShouldBeFalse(nameof(graphLockPath));
+            this.fileSystem.FileExists(graphLockPath)
+                           .ShouldEqual(this.maintenanceMode == Settings.MaintenanceMode.Git, nameof(graphLockPath));
```

Comment:
I am unfamiliar with the `ShouldEqual()` function, but take your word for the fact that it does not take two objects whose equality is ensured?

Reply:
there are two possible modes for `this.maintenanceMode`... `.Git` and `.Scalar`. So this is essentially saying "This file should exist if and only if we are in the Git mode"